tests/installation/isosize.pm

```diff
@@ -11,7 +11,12 @@ sub run {
     if (!$size || !$max || $size > $max) {
         $result = 'fail';
     }
-    bmwqemu::diag("check if actual iso size $size fits $max: $result");
+    if (!defined $size) {
```

Comment:
Well, when the path to the iso is invalid I just saw the diag line tell me

    check if actual iso size  fits XXX: false

with "XXX" being the size of the ISO that was expected. Notice the double space between 'size' and 'fits' because $size was actually "undef". It is actually *not* defined as the previous use of the ternary operator yields undef if the iso itself is undefined. That is what I am checking for. I tested, promise :-)